content/tracing/visualization/services_list.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Services list
+kind: Documentation
+further_reading:
+- link: "/tracing/setup/"
+  tag: "Documentation"
+  text: Learn how to setup APM tracing with your application
+- link: "/tracing/visualization/service"
+  tag: "Documentation"
+  text: Learn more about services in Datadog
+- link: "/tracing/visualization/resource"
+  tag: "Documentation"
+  text: Dive into your resource performances and traces
+- link: "/tracing/visualization/trace"
+  tag: "Documentation"
+  text: Understand how to read a Datadog Trace
+---
+
+## Overview
+
+After you have [instrumented your application](/tracing/setup), your reporting services appear on [the APM services page](https://app.datadoghq.com/apm/services).  
+Select a service view detailed performance insights, [read our dedicated service documentation to learn more](/tracing/visualization/service)
```

Comment:
Should there be a period after this sentence?

Reply:
👌